### ISSUE TITLE
Mention the need to have suitable browser driver in installation instructions

### ISSUE THREAD

**Issue by cscottrun** [robotframework/WebDemo#7]
I had to install geckodriver in order to pass the test. It took me a while to figure that out. Being that this tutorial is for new users, that might be a helpful note to put in the readme.

**Comment by pekkaklarck** [comment]
Good point. That ought to be mentioned in SeleniumLibrary installation instructions that are linked from README, but it would nevertheless be a good idea to mention browser drivers there explicitly.

**Comment by pekkaklarck** [comment]
Somewhat related to this, installation instructions should be changed not to use `requirements.txt`. See my comments to that here:
https://github.com/robotframework/WebDemo/pull/6#issuecomment-478339713

**Comment by aaltat** [comment]
The SelenumLibrary README has [browser driver](https://github.com/robotframework/SeleniumLibrary#browser-drivers) section, which explains installation of the browser drivers. Should we link to the documentation or copy/paste documentation here?

**Comment by pekkaklarck** [comment]
Definitely no copy-pasting, better to have more detailed explanation elsewhere. Just a note that browser specific browser driver is needed with a link to SL documentation ought to be enough.

**Comment by jabelk** [comment]
ran into the same issue for me. using `brew install geckodriver` for OS X fixed the browser driver issue.

**Comment by usmanhafeez** [comment]
i cannot find where are locators saved for username_field, and password_field. can you please tell.

**Comment by chanita** [comment]
> i cannot find where are locators saved for username_field, and password_field. can you please tell.

https://github.com/robotframework/WebDemo/blob/master/demoapp/html/index.html

**Comment by Spaceless007** [comment]
This still hasnt been fixed... Pls fix it for future users

**Comment by lblanch** [comment]
I ran into the same issue (missing geckodriver). 
For those using MacPorts, it seems their distribution of the geckodriver doesn't work in a Apple M1 architecture at the moment. I fixed it by downloading the latest driver package from the official github (https://github.com/mozilla/geckodriver/releases), which does support M1, extracting it and moving it to a folder included in my path, like '/usr/local/bin'

> Somewhat related to this, installation instructions should be changed not to use `requirements.txt`. See my comments to that here: [#6 (comment)](https://github.com/robotframework/WebDemo/pull/6#issuecomment-478339713)

Great idea! In my case, this would have helped as it would have forced me to follow the Selenium installation instructions from their webpage. When I saw the option of using 'requirements.txt' to install everything, the Selenium docs were skipped. 

Edit: to clarify, this was my first time using Python, Robot Framework or Selenium.
